stable_baselines/common/evaluation.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+
+def evaluate_policy(model, env, n_eval_episodes=10, deterministic=True,
+                    render=False, callback=None, reward_threshold=None,
+                    return_episode_rewards=False):
+    """
+    Runs policy for n episodes and returns average reward.
+    This is made to work only with one env.
```

Comment:
Curious if there's a reason for this besides ease of implementation; e.g. any source of bias from using multiple envs?

Reply:
First hunch: If you run for fixed number of episodes and episode length depends on the success/failure, this could bias towards episodes with shorter length. E.g. one worker fails bunch of CartPole episodes while another worker passes one episode, since this episode took longer. Then again, number of short/long episodes should average out to same for each worker if you run enough episodes.

Reply:
Ah yeah that would introduce a bias, well spotted!